src/Service/DigitalPostHelper.php

```diff
@@ -93,4 +198,11 @@ private function configureOptions(OptionsResolver $resolver)
             'digital_post_options',
         ]);
     }
+
+    private function getFilePath(Document $document): string
+    {
+        $this->documentUploader->specifyDirectory('/case_documents/');
+
+        return $this->documentUploader->getDirectory().'/'.$document->getFilename();
+    }
```

Comment:
We should make `DocumentUploader::getFilepath` public and use that.

Reply:
Made `DocumentUploader::getFilepath` public and use that.